web/src/index.js

```diff
@@ -129,7 +137,12 @@ class App extends Component {
 			HOMESERVER_URL = indexData.homeserver_url || HOMESERVER_URL
 			// TODO only load pack metadata when scrolled into view?
 			for (const packFile of indexData.packs) {
-				const packRes = await fetch(`${PACKS_BASE_URL}/${packFile}`, { cache })
+				let packRes;
+				if (REMOTE) {
+					packRes = await fetch(packFile, { cache })
+				} else {
+					packRes = await fetch(`${PACKS_BASE_URL}/${packFile}`, { cache })
+				}
```

Comment:
Supporting full URLs fine for `packs/index.json` too would be fine, maybe just check if `packFile` starts with `https://` and remove the `REMOTE` variable